test/BUILD.gn

```diff
@@ -364,11 +438,22 @@ test("brave_unit_tests") {
       "//brave/net/proxy_resolution/proxy_config_service_tor_unittest.cc",
     ]
 
-    deps += [ "//third_party/blink/public/common" ]
+    deps += [
+      "//brave/browser/profiles:profiles",
```

Comment:
nit : `profiles:profiles` is redundant - same with tor:tor, etc..

Reply:
Done.